docs/core/transactions.md

```diff
@@ -8,84 +8,146 @@ order: 2
+
+```bash
+simd tx send $MY_VALIDATOR_ADDRESS $RECIPIENT 1000stake
+```
 
-[`Contexts`](https://godoc.org/context) are immutable objects that contain all the information needed to process a request. In the process of creating a transaction through the `auth` module (though it is not mandatory to create transactions this way), two contexts are created: the [`Context`](../interfaces/query-lifecycle.md#context) and `TxBuilder`. Both are automatically generated and do not need to be defined by application developers, but do require input from the transaction creator (e.g. using flags through the CLI).
+#### gRPC
 
-The `TxBuilder` contains data closely related with the processing of transactions.
+[gRPC](https://grpc.io) is introduced in Cosmos SDK 0.40 as the main component for the SDK's RPC layer. The principal usage of gRPC is in the context of modules' [`Query` services](../building-modules). However, the SDK also exposes a few other module-agnostic gRPC services, one of them being the `Tx` service:
 
-+++ https://github.com/cosmos/cosmos-sdk/blob/7d7821b9af132b0f6131640195326aa02b6751db/x/auth/types/txbuilder.go#L18-L31
++++ https://github.com/cosmos/cosmos-sdk/blob/v0.40.0-rc3/proto/cosmos/tx/v1beta1/service.proto
 
-* `TxEncoder` defined by the developer for this type of transaction. Used to encode messages before being processed by nodes running Tendermint.
-* `Keybase` that manages the user's keys and is used to perform signing operations.
-* `AccountNumber` from which this transaction originated.
-* `Sequence`, the number of transactions that the user has sent out, used to prevent replay attacks.
-* `Gas` option chosen by the users for how to calculate how much gas they will need to pay. A common option is "auto" which generates an automatic estimate.
-* `GasAdjustment` to adjust the estimate of gas by a scalar value, used to avoid underestimating the amount of gas required.
-* `SimulateAndExecute` option to simply simulate the transaction execution without broadcasting.
-* `ChainID` representing which blockchain this transaction pertains to.
-* `Memo` to send with the transaction.
-* `Fees`, the maximum amount the user is willing to pay in fees. Alternative to specifying gas prices.
-* `GasPrices`, the amount per unit of gas the user is willing to pay in fees. Alternative to specifying fees.
+The `Tx` service exposes a handful of utility functions, such as simulating a transaction or querying a transaction, and also one method to broadcast transactions.
 
-The `Context` is initialized using the application's `codec` and data more closely related to the user interaction with the interface, holding data such as the output to the user and the broadcast mode. Read more about `Context` [here](../interfaces/query-lifecycle.md#context).
+An example of broadcasting a transaction is shown in [TODO](https://github.com/cosmos/cosmos-sdk/issues/7657). Please note that the `BroadcastTx` endpoint takes `TxRaw`, not bytes.
```

Comment:
is this just a personal reminder to update this as part of a future docs related PR?

Reply:
yeah exactly, I added reminders in #7657